static/mozfest/technological-suffering- and-how-mindfulness-can-help.ics

```diff
@@ -0,0 +1,16 @@
+BEGIN:VCALENDAR

```

Comment:
the file name seems to have an extra space and is therefore a duplicate of the next file